app/assets/javascripts/host_power.js

```diff
@@ -0,0 +1,61 @@
+(function(){
+
+  $(document).ready(function () {
+    powerStatus();
+  });
+
+  function hostIds() {
+    var res = []
+    $('#hosts-table input.host_select_boxes').each(function (index, item) {
+      res.push($(item).attr('id').match(/\d+/).pop());
```

Comment:
`.map` will not work here, query details get attached to the resulting array for some reason:
![](http://i.imgur.com/gFnqxfk.png)